Automattic/wp-calypso
# Adding cloudflare card to traffic settings

#### Changes proposed in this Pull Request

* This diff addresses the requirements in 401-gh-Automattic/dotcom-manage to add a card for a Cloudflare analytics tracking ID to Tools->Marketing->Traffic.

#### Testing instructions



* Navigate to http://calypso.localhost:3000/marketing/traffic/{siteUrl}?flags=cloudflare and verify that the Cloudflare card appears. (It doesn't do anything yet but it should be visible.)
* Disable the `cloudflare` feature flag temporarily by adding `?flags=-cloudflare` to the end of the URL; the card should disappear.

![image](https://user-images.githubusercontent.com/13437011/104519667-6d8cf180-55bf-11eb-9eb6-f097e4ef6a07.png)


Fixes 401-gh-Automattic/dotcom-manage
